torch/quantization/fx/_equalize.py

```diff
@@ -318,22 +414,12 @@ def maybe_get_next_input_eq_obs(node: Node, modules: Dict[str, nn.Module]) -> Op
 
     assert(node_supports_equalization(node, modules))
 
-    # Locate the following nn.ReLU or F.relu node if it exists
-    maybe_relu_node = maybe_get_next_module(node, modules, nn.ReLU)
-    if maybe_relu_node is None:
-        maybe_relu_node = maybe_get_next_module(node, modules, target_functional_type=F.relu)
-
     # Locate the following output observer if it exists.
-    # We will skip the relu node if it exists.
-    maybe_obs_node = (
-        maybe_get_next_module(node, modules, ObserverBase)
-        if maybe_relu_node is None
-        else maybe_get_next_module(maybe_relu_node, modules, ObserverBase)
-    )
+    maybe_obs_node = maybe_get_following_observer(node, modules, ObserverBase)
```

Comment:
Why do we need to find `maybe_obs_node` here? Can we directly find `maybe_get_following_observer(node, modules, _InputEqualizationObserver)`?

Reply:
The way the check current works is that it will look for the following call_module node and see if it's the specific given type. However, there should always be a quantization observer in the middle between the op_node and equalization observer, which would result in `maybe_get_following_observer(node, modules, _InputEqualizationObserver)` returning None.

So I first found the quantization observer, and then looked for the equalization observer from the quantization observer.